examples/example_experiment_all.ipynb

```diff
@@ -675,19 +673,88 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "for qubits, data in result_ghz.observations.items():\n",
-    "    print(f\"Qubits {qubits}:\")\n",
-    "    for k,v in data.items():\n",
-    "        print(f\"\\t{k}: {v}\")"
+    "# Show observations for a specific qubit layout\n",
+    "qubit_layout = [0,1,2]\n",
```

Comment:
FYI: I changed only the qubit layout in this cell

Reply:
Thanks! OK yes I wasn't sure what exactly was the update needed